shell/android/jni/TinyRenderer.cpp

```diff
@@ -100,6 +100,16 @@ void TinyRenderer::init(AAssetManager* mgr,
     vulkan::VulkanContextConfig config;
     config.terminateOnValidationError = true;
     config.requestedSwapChainTextureFormat = swapchainColorTextureFormat;
+    switch (swapchainColorTextureFormat) {
```

Comment:
Hi, while I believe you that you get different results, why do you feel this is the right fix?

can you provide screenshots of the differences just to make sure that we’re talking about the same thing?

the sRGB frame buffer might not be very well tested on Android phones